src/backend/linux/window.rs

```diff
@@ -258,15 +258,14 @@ impl From<x11::window::WindowHandle> for crate::WindowHandle {
 }
 
 impl WindowHandle {
-    #[cfg(feature = "wayland")]
-    /// Assume that this WindowHandle is from Wayland
-    pub(crate) fn unwrap_wayland(&self) -> &wayland::window::WindowHandle {
-        match self {
-            WindowHandle::Wayland(it) => it,
-            _ => unreachable!("Must use a wayland window handle"),
-        }
-    }
-
+    // #[cfg(feature = "wayland")]
+    // /// Assume that this WindowHandle is from Wayland
+    // pub(crate) fn unwrap_wayland(&self) -> &wayland::window::WindowHandle {
+    //     match self {
+    //         WindowHandle::Wayland(it) => it,
+    //         _ => unreachable!("Must use a wayland window handle"),
+    //     }
+    // }
```

Comment:
Will this need to be re-implemented later?

Reply:
I think so - it was previously used in `create_popup`, which accepts a `crate::WindowHandle` rather than a `wayland::WindowHandle`